src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -246,6 +246,165 @@ public void TestUpdateMultipleParametersForGraphQLSettings()
+
+            // Assert: Validate the RequestBodyStrict Value is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.Rest?.RequestBodyStrict);
+            Assert.AreEqual(updatedRequestBodyStrictValue, runtimeConfig.Runtime.Rest.RequestBodyStrict);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.rest.enabled {value} --runtime.rest.path {value}"
+        /// on a config with various values results in runtime config update. 
+        /// Takes in updated value for enabled and path and further 
+        /// validates whether the runtime config reflects those updated values
+        [DataTestMethod]
+        [DataRow(false, "/updatedPath", DisplayName = "Update enabled flag and path in Rest runtime settings.")]
+        public void TestUpdateMultipleParametersRestSettings(bool updatedEnabledValue, string updatedPathValue)
+        {
+            // Arrange -> all the setup which includes creating options.
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attempts to update the path value and enabled flag
+            ConfigureOptions options = new(
+                runtimeRestPath: updatedPathValue,
+                runtimeRestEnabled: updatedEnabledValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the path is updated and enabled is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.Rest?.Path);
+            Assert.IsNotNull(runtimeConfig.Runtime?.Rest?.Enabled);
+            Assert.AreEqual(updatedPathValue, runtimeConfig.Runtime.Rest.Path);
+            Assert.AreEqual(updatedEnabledValue, runtimeConfig.Runtime.Rest.Enabled);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.cache.enabled" on a config with various values results
+        /// in runtime. Takes in updated value for cache.enabled and 
+        /// validates whether the runtime config reflects those updated values
+        [DataTestMethod]
+        [DataRow(false, DisplayName = "Update enabled to be false for Cache.")]
+        [DataRow(true, DisplayName = "Update enabled to be true for Cache.")]
+        public void TestUpdateEnabledForCacheSettings(bool updatedEnabledValue)
+        {
+            // Arrange -> all the setup which includes creating options.
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
```

Comment:
should specify what's the initial/default value used in the config.

Reply:
This is not needed and not been done in any other test. The default value being mentioned here doesnt provide any significance unless we want to write a test to check if the default value is correctly applied